venona/pkg/task/task.go

```diff
@@ -31,15 +31,27 @@ const (
 	TypeAgentTask Type = "AgentTask"
 )
 
+const (
+	StatusSuccess Status = "Success"
+	StatusError   Status = "Error"
+)
+
 type (
 	// Tasks array
 	Tasks []Task
 
 	// Type of the task
 	Type string
 
+	// Task Id
+	Id string
```

Comment:
i'm not sure this is needed.
the type and status have their type definitions because they behave like enums, and have a predefined set of values.
the id is just any random string, i imagine.

Reply:
Yeah, this makes sense. Will fix it now